module/config.mjs

```diff
@@ -332,6 +334,15 @@ DND5E.weaponProficienciesMap = {
   martialR: "mar"
 };
 
+/**
+ * A mapping between `DND5E.weaponTypes` and `DND5E.attackClassifications`. Unlisted types are assumed to be
+ * of the "weapon" classification.
+ * @enum {string}
+ */
+DND5E.weaponClassificationMap = {
+  natural: "unarmed"
```

Comment:
Not sure about this. Natural weapons are considered attacks with weapons, while unarmed strikes are not.

Reply:
Okay, I can leave it blank for now until we add an actual "Unarmed" weapon type.